[CLA] Create cbusillo.md to sign cal

cbusillo opened the pull request:
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:




---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/odoo/pull/129697).

niyasraphy reviewed (commented):
hi @cbusillo ,
it seems you are signing using a different email address : cbusillo@gmail.com which is not the email you specified in the cla file. please sign using the email info@shinycomputers.com

also you can improve the commit message format as per guidelines: https://www.odoo.com/documentation/16.0/contributing/development/git_guidelines.html

thanks

cbusillo commented:
@niyasraphy thank you for the correction.  I have changed the email address in the cbusillo.md to match my signing email address.  I have also added the [CLA] tag.  Sorry about the issues!

mart-e commented:
Hello,

Thank you for the signature, it will soon be integrated into Odoo.
Future pull requests made on odoo will be validated by the runbot.
If you have existing PRs that are awaiting for CLA validation, don't forget to rebase your code as soon as this is merged.

Regards

@robodoo review+
